modules/infrastructure-node-sizing.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-host-practices.adoc
+
+[id="infrastructure-node-sizing_{context}"]
+=  Infrastructure node sizing
+
+The infrastructure node resource requirements depend on the cluster age, number of nodes and objects in the cluster as they will lead to increase in the number of metrics or time series in Prometheus. The following infrastructure node size recommendations are based on the results of cluster maximums and control plane density focused testing.
+
+[options="header",cols="3*"]
+|===
+| Number of worker nodes |CPU cores |Memory (GB)
+
+| 25
+| 4
+| 32
+
+| 100
+| 8
+| 64
+
+| 250
+| 32
+| 192
+
+| 500
+| 32
+| 192
+
+|===
+
+[IMPORTANT]
+====
+The sizing recommendations are based on the scale tests which create large number of objects across the cluster including hitting some of the cluster maximums 10000 Namespaces along with 61000 Pods, 10000 Deployments, 181000 Secrets, 400 ConfigMaps e.t.c. in case of 250 and 500 node counts on a {product-title} {product-version} cluster. Prometheus is a highly memory intensive application, the resource usage depends on various factors including the number of nodes, objects, prometheus metrics scraping interval, metrics or time series and age of the cluster while the disk size also depends on the retention period. You must take these factors in to consideration and size them accordingly.
+====
+
+[NOTE]
+====
+In {product-title} {product-version}, half of a CPU core (500 millicore) is now reserved by the system by default compared to {product-title} 3.11 and previous versions. The sizes are determined taking that into consideration.
+====
```

Comment:
s/The sizes are determined taking that into consideration./This influences the stated sizing recommendations.